tokens.json

```diff
@@ -1,5 +1,5 @@
 {
-  "Core": {
+  "GLOBAL BASE/ Core": {
```

Comment:
Do you think maybe we could go with one of `GLOBAL` or `BASE`, unless this was an agreed naming convention that I missed?

Not a hill I'm remotely close to wanting to die on though!